### PR TITLE
Add functionality to add education to profile

### PR DESCRIPTION
**Pull request by ShaneByrne0963**

### CODE REVIEW

File: profiles/templates/components/education-form.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+          <button type="submit" id="submit-education-skill"
+            class="add-item-submit absolute top-0 end-0 px-5 py-2.5 h-full text-sm font-medium text-black bg-lime-500 rounded-e-lg border border-lime-600 hover:not(disabled):bg-lime-400 focus:ring-4 focus:outline-none focus:ring-lime-300">
+            Add</button>
+        </div>
+      </div>
+    </form>
+  </div>
+  <button type="submit" form="education-form"
+    class="text-black rounded-lg bg-lime-500 py-2 px-3 border border-lime-600">Add Work
+    Experience</button>
```
Comment (stephendawsondev) [inline profiles/templates/components/education-form.html]: Issue (Blocking): This should be "Add education"

**Review by stephendawsondev — CHANGES_REQUESTED** [review]
Looks good, but I'm getting this error when trying to submit the form. I think once that is fixed and the button is updated, should be good to go. 

Ajax error

File: profiles/templates/components/education-form.html (hunk 1 above)
Comment (ShaneByrne0963) [reply profiles/templates/components/education-form.html]: Fixed

**Review by stephendawsondev — APPROVED** [review]
LGTM! 🚢

**Pull request merged**